kubernetes/actions/utils.py

```
@@ -42,4 +50,4 @@ def apply_object(apply_object: ApplyObject):
         return {"error": e.reason}
 
     except (yaml.YAMLError, json.JSONDecodeError) as e:
-        return {"error": f"Error parsing input data: {e}"}
\ No newline at end of file
+        return {"error": f"Error parsing input data: {e}"}
```

@ramibo - not your fault but swallowing an error and reporting it as a regular output is a very bad practice.
[errors are not the same as valid outputs and should be treated otherwise](https://peps.python.org/pep-0020/)